{{cookiecutter.project_name}}/Makefile

```diff
@@ -41,6 +42,9 @@ open-cov-report:
 examples:
 	@${PYTHON3} ${RUNEXAMPLES} > /dev/null && echo "Examples OK!" || echo "Examples failed!"
 
+docs html:
+	@${MAKE} -C docs html
```

Comment:
Should this be `html` or `build`. Since it won't work the first time if you don't have the requirements to build the docs. `make build` first installs the requirements.

Reply:
There could for example maybe be the following:
```sh
docs build-docs:
	@${MAKE} -C docs build

docs open-docs:
	@${MAKE} -C docs see
```
what do you think?

Reply:
Good point  
The intention is to be able to quickly build your docs from just the project directory without cding into docs. So the first time building the docs you'd do that from the /docs folder, any subsequent time you can do so from the project directory, without all the "Requirement already satisfied" spam.

Reply:
Okay, sure sounds good to me